src/process/task.rs

```diff
@@ -154,7 +162,7 @@ mod tests {
             if stat.comm == "one" && status.name == "one" {
                 found_one = true;
                 assert!(io.rchar >= bytes_to_read);
-                assert!(stat.utime >= 50, "utime({}) too small", stat.utime);
+                assert!(stat.utime >= 1, "utime({}) too small", stat.utime);
```

Comment:
This change was just to get the tests more stable, right?  (The utime accounting here is known to be a bit flakey, and I've had to fuss with these tests quite often)

Reply:
This came from #127, so I'm not sure of the original reason for it. I'll revert it.